Makefile

```diff
@@ -39,7 +39,7 @@ CONTAINER_BUILD_PARAMS = --network=host --label git_revision=${GIT_REVISION} ${C
 
 # RHCOS_VERSION should be consistent with BaseObjectName in pkg/s3wrapper/client.go
 RHCOS_BASE_ISO := $(or ${RHCOS_BASE_ISO},https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/latest/rhcos-live.x86_64.iso)
-OPENSHIFT_VERSIONS := $(or ${OPENSHIFT_VERSIONS}, $(shell cat default_ocp_versions.json | tr -d "\n\t "))
+OPENSHIFT_VERSIONS := $(or ${OPENSHIFT_VERSIONS}, $(shell ./test_ocp_versions.sh | tr -d "\n\t "))
```

Comment:
If a script is being used I think you can add the translation into it. Maybe even add a flag to the python script for a dense output.

Reply:
Not sure what you mean

Reply:
Move the `tr` operation inside the script